plugins/dell-dock/fu-dell-dock-common.h

```diff
@@ -39,9 +39,22 @@
 #define ATOMIC_HUB1_PID			    0x541A
 #define DELL_VID			    0x413C
 
+/* device IDs: ec*/
+#define DELL_DOCK_HID_VID 0x413C
```

Comment:
Also defined above (see `DELL_VID`).  I think you should reduce to one definition.

Reply:
fixed in 271ccab.